src/atlas_protocol_scripts/pp/_check_genes.py

```diff
@@ -0,0 +1,72 @@
+import anndata
+import numpy as np
+
+
+def remove_gene_version(string):
+    import re
+
+    string = str(string)
+    if string.startswith("ENSG"):
+        return re.sub(r"\..*", "", string)
+    else:
+        return string
+
+
+def append_duplicate_suffix(df, column, sep):
+    """Appends a numeric suffix to each duplicated value in the specified column based on index position."""
+    suffix_dict = {}
+    df_sorted = df.sort_values(by=column)
+    df_sorted = df_sorted.reset_index().sort_values(by="index").set_index("index")
+    for i, val in enumerate(df_sorted[column]):
+        if val in suffix_dict:
+            suffix_dict[val] += 1
+            df_sorted.at[df_sorted.index[i], column] = f"{val}{sep}{suffix_dict[val]}"
+        else:
+            suffix_dict[val] = 0
+    return df_sorted.sort_index()
+
+
+def find_unmapped_genes(adata):
+    for column in ["Gene", "symbol"]:
+        if column in adata.var.columns:
+            unmapped = adata[:, adata.var_names == "nan"].var[column].tolist()
+    return unmapped
+
+
+def aggregate_duplicate_gene_ids(adata, gene_names):
+    """
+    Collapse duplicate gene IDs in an AnnData object by summing their expression values.
+
+    Parameters
+    ----------
+        adata (anndata.AnnData): The AnnData object to process.
```

Comment:
```suggestion
    adata (anndata.AnnData): The AnnData object to process.
```

When using numpy-style docstrings (as expected in this repsitory), the parameters should not be indented.